web/src/app/lists/PaginatedList.tsx

```diff
@@ -161,16 +81,11 @@ function LoadingItem(props: { dense?: boolean }): JSX.Element {
 }
 
 export interface PaginatedListProps {
-  // cardHeader will be displayed at the top of the card
-  cardHeader?: ReactNode
-
-  // header elements will be displayed at the top of the list.
-  headerNote?: string // left-aligned
-  headerAction?: JSX.Element // right-aligned
-
   items: PaginatedListItemProps[] | CheckboxItemsProps[]
   itemsPerPage?: number
 
+  pageCount?: number
+
   isLoading?: boolean
```

Comment:
How should we be using this? I see `isLoading` as both a propType and being used in the context but not sure if it should just be one or the other

Reply:
It looks like we are passing down `loading` from Apollo a few components, but then use the context instead inside the page controls. I don't see anywhere that `isLoading` gets updated in the context though so maybe we should use the prop instead and remove `isLoading` from the context?